### DISCUSSION TITLE
cobalt2 not working

### DISCUSSION THREAD

**Discussion by DuyDangCode** [lalitmee/cobalt2.nvim#15] [Q&A]
I have installed it at lazy.lua. You can see it in below image 
![image](https://github.com/lalitmee/cobalt2.nvim/assets/115650649/f309aea2-1c34-480c-86fe-00f8abadd126)
When I switch to the Cobalt2 theme, there are no changes.
I need your help. Thank you.

**Comment by lalitmee** [comment]
Is this `LazyVim` distribution? I am not sure how to install a theme there, but I believe you should be able to install it in your custom themes. The screenshot shows that you are trying to install it directly in the spec, not sure if it will work there or not.

**Reply by DuyDangCode** [reply]
Thank you
